src/client/components/Header/index.js

```diff
@@ -0,0 +1,76 @@
+import IcomoonReact from "icomoon-react";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import React from "react";
+import iconSet from "../../../../public/icons.json";
+
+class Header extends React.PureComponent {
+  render() {
+    return (
+      <header className="header border-bottom border-secondary">
+        <Navbar className="justify-content-between" variant="dark" bg="primary">
+          <Navbar.Brand href="https://ca.gov">
+            <img
```

Comment:
i see the icons here and in the snapshot but don't see them in your PR screenshot.. why's that?

Reply:
We're leaving out the icons in the white portion of the header, at least for now! We're only including the icons in the blue portion of the header next to "Log In" and "Home".